### PR TITLE
setup the files for our project

### PR DESCRIPTION
**Pull request by drteresavasquez**
Created index.html, main.css, main.js

How to test: pull setup branch, run server, background should be blue, log should read "success"

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: aquamarine;
```
Comment (ajawashington) [inline main.css:2]: Could this be pink instead? 👀

**Review by TrinityChristiana — APPROVED** [review]
LGTM!

**Review by ajawashington — CHANGES_REQUESTED** [review]

File: main.css (hunk 1 above)
Comment (drteresavasquez) [reply main.css:2]: Ok. will update

**Review by ajawashington — APPROVED** [review]
Thank you! Looks great!

**Pull request merged**